server/TracyView_Statistics.cpp

```diff
@@ -440,6 +445,8 @@ void View::DrawStatistics()
         ImGui::Spacing();
         ImGui::SameLine();
         TextFocused( "Visible zones:", RealToString( srcloc.size() ) );
+        ImGui::SameLine();
+        copySrclocsToClipboard = ClipboardButton();
```

Comment:
Try adding some spacing or a vertical separator. This will help separate things more clearly.